packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -917,6 +920,9 @@ onMounted(() => {
 			workflowService.updateWorkflow(wf.value.dump());
 			workflowDirty = false;
 		}
+		if (canvasTransform) {
+			setLocalStorageTransform(wf.value.getId(), canvasTransform);
```

Comment:
why check if `canvasTransform` exists`, it can't be undefined

Reply:
fixed thanks!